src/js/components/Overview/Chart.tsx

```diff
@@ -30,6 +36,19 @@ const Chart = memo(({ chartConfig, data, units, id }: ChartProps) => {
           }}
         />
       );
+    case CHART_TYPE_HISTOGRAM:
+      return (
+        <Histogram
+          units={units}
+          height={CHART_HEIGHT}
+          data={data}
+          preFilter={removeMissing}
+          dataMap={translateMap}
+          onClick={(d) => {
```

Comment:
factor out this function to a constant inside `Chart` since it's reused for bar/histogram